.github/workflows/task-build-artifacts.yml

```diff
@@ -77,7 +110,15 @@ jobs:
         run: ./install_boost.sh ${{ env.BOOST_VERSION }} ${{ steps.mode.outputs.mode }}
 
       # Get Redis
-      - name: Get Redis
+      - name: Get Redis (node20)
```

Comment:
coming to think of if, why do we need Redis for build artifacts?

Reply:
because the script for the ramp file/module.json needs a redis instance to get the command list before, and the command list after loading the module in order to generate the module command list